components/oneapi-validation/src/main/java/com/wso2telco/dep/oneapivalidation/service/impl/payment/ValidateRefund.java

```diff
@@ -152,6 +157,9 @@ public void validate(String json) throws CustomException {
         } catch (JSONException e) {
             log.error("Manipulating received JSON Object: " + e);
             throw new CustomException("SVC0001", "", new String[]{"Incorrect JSON Object received"});
+        } catch (CustomException e){
+            log.error("Manipulating received JSON Object: " + e);
+            throw new CustomException(e.getErrcode(), e.getErrmsg(), e.getErrvar());
         } catch (Exception e) {
```

Comment:
[Code Review] Try to capture more specific exceptions instead of generic Exception